anta/cli/cli.py

```diff
@@ -19,12 +19,12 @@
 @click.group()
 @click.pass_context
 @click.version_option(__version__)
-@click.option('--username', show_envvar=True, default='arista', help='Username to connect to EOS', required=True)
+@click.option('--username', show_envvar=True, default='admin', help='Username to connect to EOS', required=True)
```

Comment:
so I don't really care about the default value but does it change the outputs of ythe help and so the documentation?

Reply:
No, nothing in doc.